app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -438,10 +438,14 @@ function AcuantCapture(
     const { image, cardtype, dpi, moire, glare, sharpness } = nextCapture;
     const isAssessedAsGlare = glare < glareThreshold;
     const isAssessedAsBlurry = sharpness < sharpnessThreshold;
+    const isUnsupportedDocument = cardtype !== 1;
```

Comment:
I might be mistaken, but I'm expecting `cardtype` of `0` ("None") could arise from a number of reasons the Acuant SDK wasn't able to successfully classify a document in an image, even in cases where the user is actually trying to capture a state-issued ID. Are we expecting this message to be shown in those circumstances? Just stands out to me that the messaging is rather strongly worded as assuming the document type is incorrect.

Reply:
Should we define an enum for these cardtypes? https://www.typescriptlang.org/docs/handbook/enums.html#numeric-enums

Reply:
@eileen-nava Totally possible that it's not a real concern. Have you also tested with manual uploads, i.e. those not captured through the Acuant SDK guided capture experience? I think we use Acuant SDK to process those as well.

Reply:
@aduth Yes, I tested with uploads of existing images and photos captured by the native camera.  We don't use the Acuant SDK to process their card type, glare, blurriness, etc.

@zachmargolis I think the [AcuantDocumentType](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/document-capture/components/acuant-camera.tsx#L96-L103) is sufficient.

Reply:
All we have to do is change it:
```
export enum AcuantDocumentType = {
  NONE = 0;
  ID = 1;
  PASSPORT = 2;
}
```
and then we don't need to leave comments describing what the numbers mean anymore. I think it would be a huge win

Reply:
@zachmargolis Being able to remove comments is a plus!  I did the refactor.

Reply:
Re: the concerns about the acuant SDK misidentifying an image's document type, I wrote a follow-up ticket [here](https://cm-jira.usa.gov/browse/LG-10415).